<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>981</width>
    <height>594</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>СМП</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_9">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabEvents">
      <attribute name="title">
       <string>События</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_8">
       <item row="0" column="0">
        <widget class="QSplitter" name="splitter">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <widget class="QSplitter" name="splitter_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <widget class="QTableView" name="tblEvents">
           <property name="selectionBehavior">
            <enum>QAbstractItemView::SelectRows</enum>
           </property>
          </widget>
          <widget class="QGroupBox" name="groupBox">
           <property name="title">
            <string>Фильтр:</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_4">
            <item row="1" column="0" colspan="2">
             <layout class="QGridLayout" name="gridLayout">
              <item row="0" column="1">
               <widget class="QDateEdit" name="dtFilterDate">
                <property name="enabled">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item row="0" column="0">
               <widget class="QCheckBox" name="chkFilterDate">
                <property name="text">
                 <string>Дата вызова</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item row="2" column="0" colspan="2">
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QCheckBox" name="chkFilterFIO">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="edtFilterFIO">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="inputMask">
                 <string/>
                </property>
                <property name="text">
                 <string>ФИО пациента</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item row="0" column="0" colspan="2">
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="QCheckBox" name="chkFilterNumber">
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="edtFilterNumber">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="inputMask">
                 <string/>
                </property>
                <property name="text">
                 <string>Номер</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item row="3" column="0" colspan="2">
             <layout class="QGridLayout" name="gridLayout_3">
              <item row="1" column="0">
               <widget class="QComboBox" name="cmbFilterDoneEvents">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <item>
                 <property name="text">
                  <string>Не отображать</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Только законченные</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Отображать все</string>
                 </property>
                </item>
               </widget>
              </item>
              <item row="0" column="0">
               <widget class="QCheckBox" name="chkFilterDoneEvents">
                <property name="text">
                 <string>Законченные события</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item row="4" column="1">
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>327</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="5" column="1">
             <widget class="QPushButton" name="btnFilter">
              <property name="text">
               <string>Применить</string>
              </property>
             </widget>
            </item>
            <item row="5" column="0">
             <widget class="QPushButton" name="btnGetEvents">
              <property name="text">
               <string>Получить события</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
         <widget class="QSplitter" name="splitter_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <widget class="QGroupBox" name="groupBox_2">
           <property name="title">
            <string>Информация о событии</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_5">
            <item row="0" column="0">
             <widget class="QLabel" name="lblCallNumber">
              <property name="text">
               <string>Идентификатор вызова:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLabel" name="lblClientFIO">
              <property name="text">
               <string>ФИО пациента:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QLabel" name="lblAdress">
              <property name="text">
               <string>Адрес:</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLineEdit" name="edtCallNumber">
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="edtClientFIO">
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="1" column="2">
             <widget class="QLineEdit" name="edtAddress">
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="lblCallDate">
              <property name="text">
               <string>Дата вызова:</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QLabel" name="lblSex">
              <property name="text">
               <string>Пол пациента:</string>
              </property>
             </widget>
            </item>
            <item row="2" column="2">
             <widget class="QLabel" name="lblKind">
              <property name="text">
               <string>Тип вызова:</string>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QDateEdit" name="dtCallDate">
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QLineEdit" name="edtSex">
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="3" column="2">
             <widget class="QLineEdit" name="edtCallType">
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="4" column="0">
             <widget class="QLabel" name="lblEventTime">
              <property name="text">
               <string>Время принятия вызова:</string>
              </property>
             </widget>
            </item>
            <item row="4" column="1">
             <widget class="QLabel" name="lblAge">
              <property name="text">
               <string>Полных лет:</string>
              </property>
             </widget>
            </item>
            <item row="4" column="2">
             <widget class="QLabel" name="lblCallerName">
              <property name="text">
               <string>ФИО вызывавшего:</string>
              </property>
             </widget>
            </item>
            <item row="5" column="0">
             <widget class="QTimeEdit" name="tmEventTime">
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="5" column="1">
             <widget class="QLineEdit" name="edtAge">
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="5" column="2">
             <widget class="QLineEdit" name="edtCallerName">
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="6" column="0">
             <widget class="QLabel" name="lblReceiver">
              <property name="text">
               <string>Принявший вызов:</string>
              </property>
             </widget>
            </item>
            <item row="6" column="1">
             <widget class="QLabel" name="lblContact">
              <property name="text">
               <string>Контактный телефон:</string>
              </property>
             </widget>
            </item>
            <item row="6" column="2">
             <widget class="QLabel" name="lblUrgency">
              <property name="text">
               <string>Категория срочности:</string>
              </property>
             </widget>
            </item>
            <item row="7" column="0">
             <widget class="QLineEdit" name="edtReceiver">
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="7" column="1">
             <widget class="QLineEdit" name="edtContact">
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="7" column="2">
             <widget class="QLineEdit" name="edtUrgency">
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QGroupBox" name="groupBox_3">
           <property name="title">
            <string>Работа с событием</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_2">
            <item row="0" column="0" colspan="2">
             <widget class="QLabel" name="lblResult">
              <property name="text">
               <string>Результат вызова:</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0" colspan="4">
             <widget class="QComboBox" name="cmbResult"/>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="lblNote">
              <property name="text">
               <string>Примечание:</string>
              </property>
             </widget>
            </item>
            <item row="3" column="0" colspan="4">
             <widget class="QLineEdit" name="edtNote"/>
            </item>
            <item row="4" column="2">
             <spacer name="verticalSpacer_2">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>52</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="5" column="0">
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>138</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="5" column="1" colspan="2">
             <widget class="QPushButton" name="btnAddEvent">
              <property name="text">
               <string>Добавить событие к вызову</string>
              </property>
             </widget>
            </item>
            <item row="5" column="3">
             <widget class="QPushButton" name="btnAprovedEvent">
              <property name="text">
               <string>Принять вызов</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabEvents">
      <attribute name="title">
       <string>Вызовы СМП</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_7">
       <item row="0" column="0">
        <widget class="QSplitter" name="splitter_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <widget class="QTableView" name="tblCallsSmp">
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
         </widget>
         <widget class="QGroupBox" name="groupBox_4">
          <property name="title">
           <string>Получение вызовов СМП за период</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_6">
           <item row="0" column="1" colspan="2">
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QDateEdit" name="edtCallSmpBegDate">
               <property name="enabled">
                <bool>true</bool>
               </property>
               <property name="sizePolicy">
                <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="1" column="1" colspan="2">
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <item>
              <widget class="QDateEdit" name="edtCallSmpEndDate"/>
             </item>
            </layout>
           </item>
           <item row="3" column="0" colspan="2">
            <widget class="QPushButton" name="btnCallSmpGet">
             <property name="text">
              <string>Получить вызовы</string>
             </property>
            </widget>
           </item>
           <item row="3" column="2">
            <widget class="QPushButton" name="btnCallSmpPrint">
             <property name="text">
              <string>Печать</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="lblCallDate_2">
             <property name="text">
              <string>С:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <spacer name="verticalSpacer_3">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>411</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="lblCallSmpBegDate">
             <property name="text">
              <string>По:</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
